src/Components/SoundWave/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { useRef, useEffect, useState } from "react";
+import PropTypes from "prop-types";
+
+function SoundWave({ src, waveColor, progressColor }) {
```

Comment:
Let's use an arrow function here to be consistnat with the rest of the site. 

- [ ] use arrow function